.travis.yml

```diff
@@ -1,25 +1,32 @@
 language: php
+sudo: false
 
 php:
-    - 7.0
-    - 5.6
-    - 5.5
-    - 5.4
-    - 5.3
-    - 5.3.3
+    - nightly
+    - '7.1'
+    - '7.0'
+    - '5.6'
+    - '5.5'
+    - '5.4'
+    - '5.3'
     - hhvm
     - hhvm-nightly
 
 matrix:
+    fast_finish: true
     allow_failures:
         - php: hhvm
         - php: hhvm-nightly
 
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 before_install:
     - composer self-update
```

Comment:
Not needed, done automatically by Travis (https://github.com/thephpleague/skeleton/pull/106).